v1/api/swagger/components/schemas/ProtocolParameters.yaml

```diff
@@ -7,3 +7,8 @@ properties:
     description: >-
       The ISCSI parameters for this attachment. This value must be specified if
       Protocol is iscsi
+  FC:
+    $ref: ./FCParameters.yaml
+    description: >-
+      The FC parameters for this attachment. This value must be specified if
+      Protocol is fc
```

Comment:
Be consistent with term. I would make a global change on "fc" with "FC"

Reply:
Sure will keep FC.